src/webparts/LaserficheRepositoryAccessWebPart/components/LaserficheRepositoryAccessWebPart.tsx

```diff
@@ -44,6 +45,7 @@ const FOR_MORE_INFO_VISIT = 'For more information visit';
 const ONCE_SIGNED_IN_YOULL_SEE_REPOSITORY =
   "Once signed in you'll be able to view your Laserfiche repository.";
 
+const needLaserficheSignInPage = `Missing ${LASERFICHE_SIGNIN_PAGE_NAME} SharePoint page. Please refer to the Adding App to SharePoint Site topic in the administration guide for configuration steps.`;
```

Comment:
since we use needLaserficheSignInPage in both Repository Access and Admin Configuration, can we move it to constants.ts and import it in both?